backend/report_submission/views.py

```diff
@@ -174,9 +174,9 @@ def post(self, request, *args, **kwargs):
 
             if form.is_valid():
                 general_information = sac.general_information
-                # fields = sorted(general_information.keys())
-                # for field in fields:
-                #     print(f"{field} : {general_information[field]}")
+                fields = sorted(general_information.keys())
+                for field in fields:
+                    print(f"{field} : {general_information[field]}")
```

Comment:
I'm not sure why `print` statements were brought back into the commit. They could be a `logger.debug`, but they shouldn't stay in as we loko to merge. (Although, I just noticed this is a draft PR.)